src/pages/gateway/tutorial.md

```diff
@@ -0,0 +1,204 @@
+
+## Prerequisites
+
+- An Adobe IO account to access the [Adobe Developer Console](https://developer.adobe.com/console)
+- Install [`Node.js`](https://nodejs.org/en/download/)
+  - You can run `node -v` and `npm -v` to determine if `node.js` is successfully installed.
+  
+  NOTE: Restart your IDE after installing `node`.
+
+- Install a Node Version Manager (`nvm`)
+  - (MAC) - [Node Version Manager](https://github.com/nvm-sh/nvm)
+  - (Windows) - [nvm-windows](https://github.com/coreybutler/nvm-windows)
+- Install the [Adobe `aio` CLI](https://developer.adobe.com/runtime/docs/guides/tools/cli_install/) with the following command:
+
+  ```bash
+  npm install -g @adobe/aio-cli
+  ```
+
+## Create a project and workspace
+
+1. In the [Adobe Developer Console](https://developer.adobe.com/console), select the desired organization from the dropdown in the top-right corner.
+
+1. Then click [**Create project from template**](https://developer.adobe.com/developer-console/docs/guides/projects/projects-template/).
+
+1. Select **App Builder**.
+
+1. Change the **Project title** to "tutorial".
+
+1. Click Save. You now have a workspace named "tutorial" and workspaces named "Production" and "Stage".
+
+## Install the API Mesh plugin
+
+Install the [API Mesh plugin](https://www.npmjs.com/package/@adobe/aio-cli-plugin-api-mesh) with the following command:
+
+  ```bash
+  npm install -g @adobe/aio-cli-plugin-api-mesh
+  ```
+
+## Create a mesh
+
+1. Run the following command to log into Adobe IO:
+
+  ```bash
+  aio login
+  ```
+
+1. Create a file named `mesh.json` with the following contents:
+
+    ```json
+      {
+        "meshConfig": {
+          "sources": [
+            {
+              "name": "REST",
+              "handler": {
+                "openapi": {
+                  "source": "https://accenturetest.magento.com/rest/all/schema?services=all"
+                }
+              }
+            },
+            {
+              "name": "GraphQL",
+              "handler": {
+                "graphql": {
+                  "endpoint": "https://accenturetest.magento.com/graphql"
+                }
+              }
+            }
+          ]
+        }
+      }
+    ```
+
+1. Run the following command:
+
+    ```bash
+    aio api-mesh:create mesh.json
+    ```
+
+1. Use the arrow keys to select the `tutorial` Project and click **Enter**. If you want to use this project for future operations, type `y` and press the **Enter** key.
+
+1. Use the arrow keys to select the `stage` Workspace and click **Enter**. If you want to use this workspace for future operations, type `y` and press the **Enter** key.
+
+1. Type `y` and click the **Enter** key to confirm the creation of the mesh. The mesh configuration and corresponding details will display.
+
+1. Run the `aio api-mesh:status` command. After your mesh is built, proceed to the next section.
```

Comment:
Would be good to mention that mesh provisioning could take upto 5 mins (even though it is a lot quicker than that)